packages/node/README.md

```diff
@@ -5,15 +5,15 @@ It allows you to create calls, send messages, and generate LaML responses.
 
 ## Contents
 * [Getting Started](#getting-started)
-* [REST](#setup-rest-client)
-* [LaML](#setup-laml-client)
-* [Relay](#relay)
+* [RestClient](#restclient)
+* [LaML](#laml-client)
+* [RelayClient](#relayclient)
```

Comment:
Can we reorder this README so Relay Client (and all Relay stuff) is first, and maybe document RestClient and LaML within the same "LaML" section (its not immediately obvious they are both for LaML).